docs/source/user/performance-tips.rst

```diff
@@ -172,6 +172,19 @@ Execution times are as follows, ``fastmath`` again improves performance.
 | ``do_sum_parallel_fast``|      5.37 ms    |
 +-------------------------+-----------------+
 
+In some cases you may wish to opt-in to only a subset of possible fast-math
```

Comment:
This is a nice piece of documentation, however I think it needs to go under the section entitled `Fastmath` https://github.com/numba/numba/blame/0ede0b7e8dc3ec8f7d22210573fbf268bf0d68f6/docs/source/user/performance-tips.rst#L70